### ISSUE TITLE
CarliniWagnerL2 cannot generate highly wrong classified adversarial examples

### ISSUE THREAD

**Issue by MingCheung** [tensorflow/cleverhans#981]
I use CarliniWagnerL2 to generate adversarial example for the network (32, 32, 64, 64, 200, 200), namely the network used in the paper "Towards Evaluating the Robustness of Neural Networks" by Nicholas Carlini and David Wagner.  But the model's accuracy is still very high. For untargeted attack, the accuracy is about 0.1669, for targeted attack, the accuracy is about 0.4652.
 The parameters I used are:
cwl2_params = {"confidence": 0.01, "batch_size": 128,
                          "learning_rate": 0.1, "binary_search_steps": 9,
                          "max_iterations": 10000, "abort_early": True, "initial_const": 1e-3,
                          "clip_min": 0., "clip_max": 1.}
How to generate highly wrong classified adversarial examples?

**Comment by goodfeli** [comment]
@carlini FYI

**Comment by npapernot** [comment]
Please post questions to the StackOverflow channel unless you have a reproducible bug or feature request. Thank you! https://stackoverflow.com/search?tab=newest&q=cleverhans
